### DISCUSSION TITLE
Integration with a plain langchain app ?

### DISCUSSION THREAD

**Discussion by Sinnaeve** [langflow-ai/langflow#1775] [Q&A]
Hi,

For context, in my team we already a python app which use plain Langchain for LLM orchestration, langfuse for montioring and langserve to deploy as an API. 

I was wondering if langflow is to be used as a replacement to a langchain app or if the two can work with each other (like for example: dev using langflow UI to prepare an agent which use multiple components, and then add it to our langchain app as an agent among other agents/logics).

I couldn't find documentation doing it, so may be not. But at the same time langflow seems to be built on langchain and act as user interface for it so it seems possible.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
